### PR TITLE
Add Button and Card

### PR DESCRIPTION
**Pull request by Megha-Pathak**
image
image

Added - 
Button

- Primary
- Link
- Icon Button
- Floating Action Button

Card

- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards
- Cards with shadow

### CODE REVIEW

File: Components/button.css [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+
+.btn-float-action{
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 50px;
+    height: 50px;
```
Comment (Namrata-J) [inline Components/button.css:162]: try to define height and width in rem

File: Components/button.css [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    opacity: 0;
+}
+
+
+
+.btn-toggle .btn-checkbox:checked + .btn-slider-round::before{
+    transform: translateX(-18px);
+}
+.btn-toggle .btn-checkbox:checked + .btn-slider-round{
+    background-color: #66263f;
```
Comment (Namrata-J) [inline Components/button.css:199]: you could have used color variable here

File: Components/cards.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
+.card {
+    margin: 1rem 2rem;
+    max-width: 300px;
+    position: relative;
+    background-color: #fff;
```
Comment (Namrata-J) [inline Components/cards.css:5]: here as well

File: Components/cards.css [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    display: flex;
+    max-width: 500px;
+    gap: 1rem;
+    background-color: #fff;
+    padding: 1rem;
+    border-radius: 0.4rem;
+  }
+  
+  .card-horizontal-img {
+    max-width: 200px;
```
Comment (Namrata-J) [inline Components/cards.css:115]: try using rems for your heights and widths and also color variables, rest of your code seems good to me

**Review by Namrata-J — APPROVED** [review]

File: Components/button.css [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+}
+.link-no-style{
+    text-decoration: none;
+    color: inherit;
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    padding: 0;
+}
+button.link-no-style{
```
Comment (dev-enforced) [inline Components/button.css:25]: You can remove the button element here.If you intend to use it then you can simply pass this class in the HTML file

File: Components/button.css [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+.btn-toggle .btn-checkbox:checked + .btn-slider-round{
+    background-color: #66263f;
+}
+
+
+
+
+
+
+
```
Comment (dev-enforced) [inline Components/button.css:207]: Can remove these extra spaces here

File: Components/button.css [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+    border-radius: 50%;
+    color:var(--complimentary-text-color);
+    background-color: var(--primary-color);
+    z-index: var(--z-index-3);
+    position: fixed;
+    bottom: 1rem;
+    right: 1rem;
+}
+
+
```
Comment (dev-enforced) [inline Components/button.css:173]: Here too

**Review by dev-enforced — APPROVED** [review]
The code looks good. Well structured CSS classes and image demonstration also looks good:
image
The link wasn't working correctly
Good work @Megha-Pathak

**Pull request merged**